airflow/ui/src/pages/Dashboard/HistoricalMetrics/MetricsBadge.tsx

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Badge } from "@chakra-ui/react";
+
+type MetricBadgeProps = {
+  readonly color: string;
+  readonly runs: number;
+};
+
+export const MetricsBadge = ({ color, runs }: MetricBadgeProps) => (
+  <Badge bg={color} borderRadius={15} minWidth={10} px={5} py={2} size="lg">
```

Comment:
```suggestion
  <Badge bg={color} borderRadius={15} minWidth={10} px={4} py={1} size="md">
```

We can make this all smaller.